src/ToDoList.js

```diff
@@ -11,90 +11,163 @@ let taskData = [
-            <img
-              onClick={handleAddTask}
-              src={AddTasksBtn}
-              alt="next button"
-              width="40"
-              height="40"
-            />{" "}
-          </a>
-        }
-        modal
-        open={isOpen}
-      >
-        {(close) => (
-          <div className="model">
-            <div className="row-1">
-              <span style={{ flexGrow: "2" }}>Add Task </span>
-              <button onClick={() => close()}>
-                <span aria-hidden="true">×</span>
-              </button>
-            </div>
-            <form onSubmit={handleAddTaskSubmit}>
-              <div className="add-task-inputs">
-                <input
-                  type="text"
-                  className="form-control input-text"
-                  placeholder="Task Name"
-                  required=""
-                />
-                <textarea
-                  type="text"
-                  className="form-control input-text"
-                  placeholder="Task Description"
-                  required=""
-                />
-                <button type="submit" className="submit-new-task">
-                  <span aria-hidden="true">Submit</span>
+    <>
+      <div className="to-do-tasks">
+        <div style={{ fontSize: "25px", display: "inline-block" }}> TO-DO:</div>
+        <Popup
+          contentStyle={{ minWidth: "30%", width: "30%" }}
+          trigger={
+            <a className="add-task" href="#">
+              <img
+                onClick={handleAddTask}
+                src={AddTasksBtn}
+                alt="next button"
+                width="40"
+                height="40"
+              />{" "}
+            </a>
+          }
+          modal
+          open={isOpen}
+        >
+          {(close) => (
+            <div className="model">
+              <div className="row-1">
+                <span style={{ flexGrow: "2" }}>Add Task </span>
+                <button onClick={() => close()}>
+                  <span aria-hidden="true">×</span>
                 </button>
               </div>
-            </form>
+              <form onSubmit={handleAddTaskSubmit}>
+                <div className="add-task-inputs">
+                  <input
+                    type="text"
+                    className="form-control input-text"
+                    placeholder="Task Name"
+                    required=""
+                  />
+                  <textarea
+                    type="text"
+                    className="form-control input-text"
+                    placeholder="Task Description"
+                    required=""
+                  />
+                  <button type="submit" className="submit-new-task">
+                    <span aria-hidden="true">Submit</span>
+                  </button>
+                </div>
+              </form>
+            </div>
+          )}
+        </Popup>
+        {taskList
+          .filter((e) => e.isComplete !== true)
```

Comment:
```suggestion
          .filter((taskItem) => taskItem.isComplete !== true)
```
Use descriptive variable names.